content/microcontrollers/esp8266-d1mini.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "ESP8266 - d1mini"
+weight: 3
+---
+
+The [Espressif ESP8266](https://www.espressif.com/en/products/socs/esp8266) d1mini is a very small yet powerful SoC that is usually used for WiFi applications thanks to its built-in radio.
+
+## Interfaces
+
+| Interface | Hardware Supported | TinyGo Support |
+| --------- | ------------- | ----- |
+| GPIO      | YES | YES |
+| UART      | YES | YES |
+| SPI      | YES | Not Yet |
+| I2C      | YES | Not Yet |
```

Comment:
The ESP8266 doesn't support I2C, it is implemented mostly in software in the official SDK.